test/Module.bazel

```diff
@@ -0,0 +1,40 @@
+module(
```

Comment:
not sure if casing matters but this file should be `MODULE.bazel`, not `Module.bazel`

Reply:
Done